servicebroker/test/config/setting_test.js

```diff
@@ -0,0 +1,91 @@
+'use strict';
+
+var path = require('path');
+var expect = require('chai').expect;
+var setting = {
+  "port": "testPort",
+  "username": "username",
+  "password": "password",
+  "dbUri": "postgress://postgres@server:port/dbname",
+  "apiServerUri": "http://apiserveruri"
+}
+describe('config setting Test Suite', function() {
+  it('Should get port correctly', function() {
+    var settingObj = require(path.join(__dirname, '../../lib/config/setting.js'))(setting);
+    expect(settingObj.port).to.equal(setting.port);
+  });
+  it('Should get username and password correctly', function() {
+    var settingObj = require(path.join(__dirname, '../../lib/config/setting.js'))(setting);
+    expect(settingObj.username).to.equal(setting.username);
+    expect(settingObj.password).to.equal(setting.password);
+  });
+  context('Parse dbUri', function() {
+    it('Should parse dbUri correctly', function() {
+      var dbSetting = require(path.join(__dirname, '../../lib/config/setting.js'))(setting).db;
+      expect(dbSetting.uri).to.equal(setting.dbUri);
+      expect(dbSetting.server).to.equal("postgress://postgres@server:port");
+      expect(dbSetting.name).to.equal("dbname");
+    });
+
+    it('Should filter the last slash when parse dbUri', function() {
+      var settingTmp = {};
```

Comment:
why not create setting in a beforeEach? This would just be changing the dbUri before invoking the require